### ISSUE TITLE
Fluent Icons Support

### ISSUE THREAD

**Issue by AathifMahir** [microsoft/ProjectReunion#254]
Currently, Microsoft Fluent Icons Supports Android and iOS but not Windows Yet, Would love to see Fluent Icons makes the way to Windows app with the Support for UWP, Win32, etc..

For making the UI consistent with icons.

# Proposal: [Microsoft Fluent Icons Nuget] 

## Summary

Microsoft Fluent Icons Support across the Windows Platforms.
## Rationale

* Fluent Icons is already available on Third-party ecosystem but not First-Party one.
* Fluent Icons should be supported Windows at launch but it didn't.

## Scope

| Capability | Priority |
| :---------- | :------- |
| Allows Developers to be consistent on icons across the ecosystem | Must |
| Flexibility on picking the icons without wasting much time | Should |
| This proposal will allow developers to accomplish needs based on client systems | Could |
| This proposal will allow end users to have consistent design across-the-board | Could |

**Comment by ShankarBUS** [comment]
WinUI's repo would be a more appropriate place for UI related things like this.

**Comment by AathifMahir** [comment]
> WinUI's repo would be a more appropriate place for UI related things like this.

Yeah, I know but this related to every windows platform that includes UWP, Win32, Etc..

Therefore Project Reunion is appropriate place as far as i know

**Comment by ShankarBUS** [comment]
**WinUI 3** is one of **ProjectReunion's** component! It is the UI framework for both the **UWP** & **Win32** platforms!

There is already an issue regarding this on WinUI's repo - https://github.com/microsoft/microsoft-ui-xaml/issues/2439

**Comment by stevewri** [comment]
Agreed: a WinUI solution would be deployable in both UWP and Win32 apps (provided the Win32 app adopted XAML UI). Moving to WinUI repo.
